.travis.yml

```diff
@@ -1,11 +1,14 @@
 language: node_js
 node_js:
-  - "9"
   - "8"
+  - "10"
 sudo: false
 cache:
   directories:
     - node_modules
+install:
+  - npm install
+  - npm install codecov
```

Comment:
suggestion: remove `install` and move the `npm install codecov` line to `after_success`